common/src/sql/dbinit.sql

```diff
@@ -1573,18 +1573,13 @@ CREATE TABLE omicron.public.system_update (
     -- this resource different from every other resource for little benefit.
 
     -- Unique semver version
-    version STRING(64) NOT NULL, -- TODO: length
-    -- version string with maj/min/patch 0-padded to be string sortable
-    version_sort STRING(64) NOT NULL -- TODO: length
+    version STRING(64) NOT NULL -- TODO: length
```

Comment:
One random thing here... I noticed a lot of other strings are defined as 63 characters in the DB and I'm not exactly sure why that is.

Reply:
It seems like CRDB itself imposes that length limit on some of its internal strings: https://www.cockroachlabs.com/docs/stable/schema-design-overview.html#hard-limits

Reply:
Oh yeah, there's a comment at the top but it is ambivalent about 63 or 64.

https://github.com/oxidecomputer/omicron/blob/302a7307239827e5ac1ca37efafc0649d8f9f003/common/src/sql/dbinit.sql#L11-L16